modules/platforms/dotnet/Apache.Ignite/Apache.Ignite.csproj

```diff
@@ -56,7 +56,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\README.md" Pack="true" PackagePath="\"/>
+    <None Include="..\README.md" Pack="true" PackagePath="\" />
```

Comment:
Was this change automatic?

Reply:
Yes, this line was added manually previously, but my IDE reformatted the file now. Let's leave it - keep the IDE happy :)